include/libtorrent/torrent_handle.hpp

```diff
@@ -1265,6 +1281,24 @@ namespace aux {
 		// client builds.
 		std::shared_ptr<torrent> native_handle() const;
 
+		// Returns true if the torrent is active in its session; returns false if
```

Comment:
I think this sentence is a bit misleading. The torrent doesn't have to be "active". It just have to *not* have been removed